src/main/java/com/company/managementservice/model/dto/SalaryDto.java

```diff
@@ -0,0 +1,33 @@
+package com.company.managementservice.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SalaryDto {
+
+    @JsonProperty("amount")
+    private Integer amount;
+
+    @JsonProperty("currency")
+    private String firstName;
+
+    @JsonProperty("from_date")
+    private LocalDate fromDate=LocalDate.now();
+
+    @JsonProperty("to_date")
+    private LocalDate toDate;
+
+    @JsonProperty("is_current")
+    private Boolean isCurrent;
+
+
+}
```

Comment:
Format the code. Command + A, Command + option + L . Do this in all the files

Reply:
Okay will push the formatted one